### ISSUE TITLE
Viable engines for Aircraft

### ISSUE THREAD

**Issue by perspicacious-slinki** [nrlulz/ACF#58]
So basically, when it comes to ACF fin aircraft, you need an awful lot of power and torque for the weight of the engine for satisfactory performance.

With normal 25% fuel, options start to limit themselves, especially in higher engine size classes. There just isn't much in the way of high power/torq, low weight engines. There also isn't really a way to implement this easily without totally screwing tank balance, but requiring fuel could be a start.

TLDR; big(~20l) racing engines for planes plz.
